### DISCUSSION TITLE
Generate custom web fonts for admin

### DISCUSSION THREAD

**Discussion by malyMiso** [shopsys/shopsys#2230] [Q&A] [answered]
I want to add custom web fonts from SVG icons for admin in a project build on top of SSFW 9.1. 

Web fonts are generated by webpack. As a source for admin SVG icons is used SSFW node module directory. I can't find a way how to generate custom web fonts and add them to existing ones. 

Is there any way how to achieve this ?

**Comment by grossmannmartin** [comment, marked as answer]
`webpack.config.js` is part of the project, so you're able to change the source of the SVG's for generating webfont. 
So, you can probably change the source to a directory of your choosing. 

But if you want to keep the current admin SVG, you will need to copy current icons from node_modules to your folder.

**Reply by malyMiso** [reply]
Thanks for the reply.  You are right, it works this way.

The reason I am asking is that I would like to keep SSFW SVGs (also up to date) along with custom SVGs.

I am looking for a way how I can keep the current admin SVGs (without copying) and easily add new SVGs, or overwrite existing ones from SSFW.

Thus, I will try to implement function for merging SSFW's SVGs and project's SVGs to one direcotry, before web fonts are generated
